package/yast2-storage-ng.changes

```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Wed Apr 22 13:29:05 CEST 2020 - aschnell@suse.com
+
+- disallow to edit LVM cache pools
```

Comment:
The OBS check scripts might want a bug or feature number.

Reply:
Trello says this is for
https://bugzilla.suse.com/show_bug.cgi?id=1099744

Reply:
It is remotely related to that bug. I have added the number.